src/frontend/pages/chat/chat.js

```diff
@@ -391,11 +390,7 @@ export class Chat extends LitElement {
   }
 
   focusOnEditor(event) {
-    this.inputControlsRef.value?.editorRef.value.textareaRef.value.focus();
-
-    // Bug da fixare, segnalato in proposal
-    // if(event.detail.eventType === "keyPressed")
-    // this.inputControlsRef.value.editorRef.value.textareaRef.value = ""
+    this.inputControlsRef.value?.editorRef.value.editorRef.value.focus();
```

Comment:
In `input-controls.js` c'è un metodo che si chiama focusEditor, che sostanzialmente fa quello che c'è qui in maniera più pulita.
Mettere quello.